TutorLizard.Web/Views/Tutor/CreateAd.cshtml

```diff
@@ -52,15 +55,16 @@
                     <input class="form-check-input" asp-for="IsRemote" /> @TutorLizard.Web.Resources.Tutor.IsRemote
                 </label>
             </div>
-            
+    </div>
+        <div class="col-md-6">
             <div class="form-group">
-                <label asp-for="Description" class="control-label">@TutorLizard.Web.Resources.Tutor.Description</label>
-                <textarea asp-for="Description" class="form-control" placeholder="@TutorLizard.Web.Resources.Tutor.Description"></textarea>
+                <label asp-for="Description" class="control-label"></label>
+                <textarea asp-for="Description" class="form-control" rows="12"></textarea>
```

Comment:
If you keep the submit button in the 2nd column instead of below the whole form, I like 11 rows better than 12, because it leaves less awkward spacing in the 1st column.
However, I can let it be, so if you don't feel like changing it, just mark this as resolved.

Reply:
ah, it supposed to be under both rows. Changed it!